gql/transport/data_structures/graphql_request.py

```diff
@@ -0,0 +1,37 @@
+from typing import Any, Dict, Optional
+
+from attr import dataclass
```

Comment:
Why don't you import `dataclass` from `dataclasses` instead of `attr`?
At the current version we should support Python from version 3.7

Reply:
agree, vscode autocompleted that for me 😷